scripts/update_locks.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+
+external_apko=$(bazel cquery @rules_apko//apko:resolved_toolchain --output=files)
```

Comment:
is there an error message to tell the user they ought to run this script?

Reply:
unfortunately no. I shall add some docs. for end users, we will change the error message to suggest that they should run this command.